docker/Dockerfile.test

```diff
@@ -0,0 +1,21 @@
+FROM ottertune-base
+
+ENV DJANGO_SETTINGS_MODULE=website.settings
+ENV C_FORCE_ROOT=true
+
+RUN mkdir -p /app
```

Comment:
I do not include those lines in the base image. My goal with the base image was to separate the package installation from the application setup since installing all of the apt/pip packages takes a long time.